project/ws/app/src/lib/routes/app-toc/components/app-toc-banner/app-toc-banner.component.html

```diff
@@ -101,30 +101,43 @@ <h3 class="overview-description mat-body-1" *ngIf="showSubtitleOnBanner" i18n-ti
 </div>
 <ng-container *ngIf="content?.primaryCategory === primaryCategory.BLENDED_PROGRAM">
   <div class="batch-div">
-    <ng-container *ngIf="content" [ngTemplateOutlet]="batchDropdown"></ng-container>
-    <ng-container *ngIf="batchData && !batchData?.enrolled  && batchData?.workFlow?.wfInitiated &&
-      (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.REJECTED) && (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.REMOVED) &&
-      (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.WITHDRAWN) &&
-      (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.APPROVED) && batchData?.content?.length">
-      <p class="margin-remove-bottom mat-body-1 padding-m bg-white">
-        {{batchData?.workFlow?.batch.name}} - ({{batchData?.workFlow?.batch.startDate | date: 'dd-MM-yyyy'}} - {{batchData?.workFlow?.batch.endDate | date: 'dd-MM-yyyy' || 'present'}})
-      </p>
-      <ng-container>
-        <button class="withdraw-batch" (click)="!disableWithdrawnBtn && requestToWithdrawDialog()" [disabled]="disableWithdrawnBtn" [ngClass]="{ 'disable-btn': disableWithdrawnBtn }" >
-            <ng-container i18n> {{ 'apptocbanner.withDrawYourRequest' | translate }} </ng-container>
-        </button>
+    <div class="request-block">
+        <ng-container *ngIf="content" [ngTemplateOutlet]="batchDropdown"></ng-container>
+      <ng-container *ngIf="batchData && !batchData?.enrolled  && batchData?.workFlow?.wfInitiated &&
+        (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.REJECTED) && (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.REMOVED) &&
+        (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.WITHDRAWN) &&
+        (batchData?.workFlow?.wfItem?.currentStatus !== WFBlendedProgramStatus.APPROVED) && batchData?.content?.length">
+        <p class="margin-remove-bottom mat-body-1 padding-m bg-white">
+          {{batchData?.workFlow?.batch.name}} - ({{batchData?.workFlow?.batch.startDate | date: 'dd-MM-yyyy'}} - {{batchData?.workFlow?.batch.endDate | date: 'dd-MM-yyyy' || 'present'}})
```

Comment:
can we put option check for batch also  "batchData?.workFlow?.batch.name"  same for other instance in this line